nixos/modules/tasks/encrypted-devices.nix

```diff
@@ -56,6 +56,13 @@ in
   };
 
   config = mkIf anyEncrypted {
+    assertions = map (dev: {
+      assertion = dev.label != null;
```

Comment:
Correct me if I'm wrong, but shouldn't be **dev.encrypted.label** instead?

Reply:
Good catch! #29926